src/wp-includes/rest-api/endpoints/class-wp-rest-font-faces-controller.php

```diff
@@ -874,14 +874,8 @@ protected function handle_font_file_upload( $file ) {
 
 		$overrides = array(
 			'upload_error_handler' => array( $this, 'handle_font_file_upload_error' ),
-			// Arbitrary string to avoid the is_uploaded_file() check applied
-			// when using 'wp_handle_upload'.
-			'action'               => 'wp_handle_font_upload',
 			// Not testing a form submission.
 			'test_form'            => false,
-			// Seems mime type for files that are not images cannot be tested.
-			// See wp_check_filetype_and_ext().
-			'test_type'            => true,
 			// Only allow uploading font files for this request.
 			'mimes'                => WP_Font_Utils::get_allowed_font_mime_types(),
 		);
```

Comment:
```suggestion
		);

		// Bypasses is_uploaded_file() when running unit tests.
		if ( defined( 'DIR_TESTDATA' ) && DIR_TESTDATA ) {
			$overrides['action'] = 'wp_handle_mock_upload';
		}
```

I'm surprised this belongs here; but this is what the Attachment controller does to work with unit tests:
https://github.com/WordPress/wordpress-develop/blob/45ec568f0ce905b9aabfb469b613e39d042ad3b7/src/wp-includes/rest-api/endpoints/class-wp-rest-attachments-controller.php#L1283-L1286

Reply:
Done in d0aff8a7e3fad9c522a436f63138f88ccb21112b

> I'm surprised this belongs here

I am too, I spent some time looking for an appropriate filter but, alas, it doesn't exist.